lib/cosmos/conversions/conversion.rb

```diff
@@ -16,11 +16,14 @@ class Conversion
     attr_reader :converted_type
     # @return [Integer] The size in bits of the converted value
     attr_reader :converted_bit_size
+    # @return [Integer] The size in bits of the converted array value
+    attr_reader :converted_array_size
```

Comment:
What is this for? Why is it part of this change?

Reply:
It is for DART (as is this change).  Largely unrelated, but needed to fully defined converted value types.